.buildkite/pipeline.nix

```diff
@@ -54,6 +54,14 @@ in
       '';
     };
 
+    scalafmt = commonAttrs // {
+      dependsOn = [ compile ];
```

Comment:
Why not other way around? Checking formatting is much more lightweight than compiling.

Reply:
Nice idea, changed, and now compile depends on formatting.